### DISCUSSION TITLE
can i subscribe like this if i want to target a field not record id?

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#3551] [Q&A]
```
pb.collection('auction_info').subscribe('catalog="zstx1yjw1kfhjyh"', function (e) {
                console.log(e.action);
                console.log(e.record);
            });
```

**Comment by ganigeorgiev** [comment]
No. The topic can be either wildcard (`*`) or a record id. You can perform the filtering inside the callback (aka. `if (e.record.catalog == "...") { ... }`).

Filter, expand and other request query parameters support for the realtime subscriptions are planned for the next release but there are no ETAs yet.
